README.md

```diff
@@ -1,4 +1,61 @@
 # meta
-Meta issues concerning many/all the Plone repositories.
 
-See the [Issues](https://github.com/plone/meta/issues) for details.
+This repository's aim is to define common configuration files
+across all/most repositories in the [GitHub plone organization](https://github.com/plone).
+
+This allows Plone developers and contributors
+to have the same expectations/workflow
+when contributing to any of the Plone repositories.
+
+The following areas are configured on repositories
+where the configuration has been applied.
+
+## Testing
+
+A `tox.ini` configuration is added to each repository to allow contributors
+to test their contributions, as well as running the tests on GitHub Actions.
+
+To run the tests, run the following commands:
+
+```shell
+python3.11 -m venv venv
+. venv/bin/activate
+pip install tox
+tox -e test
+```
+
+Note: the GitHub Actions workflow is __not__ installed on each repository,
+but rather shared by all repositories.
+
+The workflow definition can be found in this repository in `shared-workflows/tests.yml`.
+
+## Code analysis / formatting
+
+On the `tox.ini` configuration there are two environments
+to format and run quality check tools:
+
+```shell
+tox -e format
+tox -e lint
+```
+
+Note: they are based on `pre-commit`.
+Each repository needs to be enabled on [pre-commit.ci](https://pre-commit.ci/)
+for the formatting/linting to run on each commit.
+
```

Comment:
Maybe add a few paragraphs:

Note that pre-commit is smart: it only checks the files that you touch.

Not everyone like pre-commit hooks, and this is fine. We have configuration, but no hooks are installed by default. If you _do_ want to automatically run pre-commit when you make a commit, you can:

```
pre-commit install --install-hooks
```

Reply:
I made it a bit shorter and to the point, but I hope is enough 😄 🍀